tests/test_task_post.py

```diff
@@ -0,0 +1,82 @@
+from rest_framework import status
+from .common import EndpointTestCase
+
+
+class TestTaskPost(EndpointTestCase):
+    """
+    Tests for '/api/students/<int:student_id>/tasks/' [POST] endpoint.
+    """
+
+    # Fixture specific URL to available student data.
+    VALID_URL = "/api/students/2/tasks/"
+    # Fixture specific URL to student data not available for current user.
+    NOT_PERMITTED_URL = "/api/students/1/tasks/"
+    # Fixture specific URL to invalid student ID.
+    STUDENT_NOT_FOUND_URL = "/api/students/12345/tasks/"
+    DATA_JSON_POST = {
+        "name": "Nowe zadanie do zrobienia|Another new task to do",
```

Comment:
-> `Another new task to do` is enough :)

Reply:
Sorry I will repair that